aristanetworks/ansible-avd
# Feat(eos_designs): Add local user ssh key support

## Change Summary

Exposes the existing single ssh public key support in eos_cli_config_gen to eos_designs by generating the structured config for it.

## Component(s) name

`arista.avd.eos_designs`

## How to test

In eos_designs group vars, define a `local_users: username: ssh_key:` string and verify that it emits the same in the structured config for eos_cli_config_gen to consume. No line is emitted when not defined.

I suspect that this warrants a molecule test update, but Molecule is totally new to me. Can someone hold my hand/point me at documentation for going through updating and running `ansible_collections/arista/avd/molecule/eos_designs_unit_tests/` to cover this feature?

## Checklist

### Repository Checklist
